freqtrade/exchange/__init__.py

```diff
@@ -158,9 +158,9 @@ def id(self) -> str:
         """exchange ccxt id"""
         return self._api.id
 
-    def klines(self, pair: str) -> DataFrame:
+    def klines(self, pair: str, copy=True) -> DataFrame:
         if pair in self._klines:
-            return self._klines[pair].copy()
```

Comment:
would you mind adding a test to "confirm" this - i think you can do this easily by using 
```
assert id(exchange.klines('pair') != id(exchange.klines('pair')
assert id(exchange.klines('pair') != id(exchange.klines('pair', copy=True)
assert id(exchange.klines('pair', copy=True) != id(exchange.klines('pair', copy=True)
```
(note that the above is mock-code, so syntax-errors may have occured...

Reply:
"is" and ''is not" do the job I think ...